ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/LedgerDB/OnDisk.hs

```diff
@@ -1189,7 +1198,8 @@ shrinker _ (At cmd) =
     case cmd of
       Current      -> []
       Push _b      -> []
-      Snap         -> []
+      Snap         -> [At Flush]
```

Comment:
Hmm, I'm not sure about this. Specially now that we removed the flush operation before taking a snapshot.